jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibExtensionTest.java

```diff
@@ -85,6 +86,21 @@ public void testTo() {
     Assert.assertEquals("some cred helper", testJibExtension.getTo().getCredHelper());
     Assert.assertEquals("some username", testJibExtension.getTo().getAuth().getUsername());
     Assert.assertEquals("some password", testJibExtension.getTo().getAuth().getPassword());
+
+    // Testing scenario where jib.to.tags is set to a collection containing a null value.
+    HashSet<String> tags = new HashSet<String>();
+    tags.add(null);
+    tags.add("tags1");
+    try {
+      testJibExtension.to(
+          to -> {
+            to.setTags(tags);
+          });
+      testJibExtension.getTo().getTags();
+      Assert.fail("Expect this to fail");
```

Comment:
Nit: if the message doesn't provide additional helpful information or details, just use `Assert.fail()`.

And I think it's meaningful to move `testJibExtension.to()` out of the try-catch block.

Reply:
Ah yes, having all the extra lines outside the try-catch makes sense. Thanks!